.github/workflows/deploy-workflow.yml

```diff
@@ -95,3 +92,43 @@ jobs:
       - name: Deploy to Google Cloud Storage
         run: gsutil -m cp -r . gs://${{ secrets.BUCKET_NAME }}
         working-directory: ./autoscheduler/static
+
+  run-migrations:
+    name: Run Migrations
+    runs-on: ubuntu-latest
+    env:
+      GCP_DB_NAME: ${{ secrets.GCP_DB_NAME }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Fill in Postgres credentials
+        run: bash fill_credentials.sh ${{ secrets.POSTGRES_USER }} ${{ secrets.POSTGRES_PASS }}
+
+      # Don't need this
+      - name: Fill .env Google Auth credentials
+        run: bash fill_google_auth_credentials.sh ${{ secrets.GOOGLE_CLIENT_ID }} ${{ secrets.GOOGLE_CLIENT_SECRET }}
+
+      # Don't need this

```

Comment:
What's with these comments? It seems like at least `fill_credentials` should be necessary to make migrations, and if you don't need the step what's preventing you from removing it

Reply:
lol yeah I totally forgot to remove them